indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -339,6 +339,17 @@ public static Configuration injectSystemProperties(Configuration conf)
     return conf;
   }
 
+  public static Configuration injectSystemAndJobProperties(Configuration conf, HadoopDruidIndexerConfig config)
+  {
+    Configuration updatedConf = injectSystemProperties(conf);
+    HadoopIngestionSpec schema = config.getSchema();
+
+    for (final Map.Entry<String, String> entry : schema.getTuningConfig().getJobProperties().entrySet()) {
+      updatedConf.set(entry.getKey(), entry.getValue());
+    }
```

Comment:
jobProperties is owned by HadoopDruidIndexerConfig and provides addJobProperties(job) to do this.

most call sites do following for Job..
```
JobHelper.injectSystemProperties(job);
config.addJobProperties(job);
```

could you do same instead of adding this method and may be you need to add a HadoopDruidIndexerConfig.addJobProperties(conf) like we have two overloaded versions of JobHelper.injectSystemProperties() . thanks.